src/ws/ws-server.js

```diff
@@ -217,13 +219,25 @@ function processNewConnection(ws) {
         });
 }
 
+function createNewParticipant(message) {
+    return UserService.create(message.userData)
+              .then((user) => {
```

Comment:
As agreed - it user service operation will never throw but the error fact is 'null' in then callback. Please add a check for `user `